common/management/commands/create_test_data.py

```diff
@@ -0,0 +1,1775 @@
+        )
+        metadata_sets.append(reservation_metadata_set)
+
+    metadata_sets = ReservationMetadataSet.objects.bulk_create(metadata_sets)
+
+    metadata_fields = {field.field_name: field for field in _create_metadata_fields()}
+
+    field_combinations = [
+        FieldCombination(
+            supported=[
+                "reservee_first_name",
+                "reservee_last_name",
+                "reservee_email",
+                "reservee_phone",
+                "reservee_address_street",
+                "reservee_address_zip",
+                "reservee_address_city",
+                "name",
+            ],
+            required=[],
+        ),
+        FieldCombination(
+            supported=[
+                "reservee_type",
+                "reservee_first_name",
+                "reservee_last_name",
+                "reservee_organisation_name",
+                "reservee_phone",
+                "reservee_email",
+                "reservee_id",
+                "reservee_is_unregistered_association",
+                "reservee_address_street",
+                "reservee_address_city",
+                "reservee_address_zip",
+                "billing_first_name",
+                "billing_last_name",
+                "billing_phone",
+                "billing_email",
+                "billing_address_street",
+                "billing_address_city",
+                "billing_address_zip",
+                "home_city",
+                "purpose",
+            ],
+            required=[
+                "reservee_first_name",
+                "reservee_last_name",
+                "reservee_address_street",
+                "reservee_address_city",
+                "reservee_address_zip",
+            ],
+        ),
+        FieldCombination(
+            supported=[
+                "reservee_type",
+                "reservee_first_name",
+                "reservee_last_name",
+                "reservee_organisation_name",
+                "reservee_phone",
+                "reservee_email",
+                "reservee_id",
+                "reservee_is_unregistered_association",
+                "reservee_address_street",
+                "reservee_address_city",
+                "reservee_address_zip",
+                "billing_first_name",
+                "billing_last_name",
+                "billing_phone",
+                "billing_email",
+                "billing_address_street",
+                "billing_address_city",
+                "billing_address_zip",
+                "home_city",
+                "age_group",
+                "applying_for_free_of_charge",
+                "free_of_charge_reason",
+                "name",
+                "description",
+                "num_persons",
+                "purpose",
+            ],
+            required=[
+                "purpose",
+            ],
+        ),
+    ]
+
+    field_combinations_loop = cycle(field_combinations)
```

Comment:
**Metadata**: are the sets sufficient or should they be more like in the dev/test/stage/prod environments (the specific "form" combinations)? @joonatank

Reply:
I showed the configured field combinations to @joonatank previously, though if there are some more "common" combinations, it would benefit us to add those here.

Reply:
The combinations can be seen in Django admin of all environments (dev has some others too); they are basically:
- **Lomake 1**: name and email + phone is asked (mandatory), use on reservation units for individual users only
- **Lomake 2**: adds description (req) and number of persons (opt), plus the fields for other customer types (all required but reservee_type + is_unreg... will be taken into account)
- **Lomake 3**: adds home_city and (reservation) purpose (both req.) and reservation name (opt, the shorter description); these two forms used on reservation units which require more information, possibly in order to make a decision whether to confirm or deny a reservation which requires staff handling
- **Lomake 4**: adds age_group to the above (req)
- the variants of 3 & 4 ("alennus sallittu") add applying_for_free_of_charge; for use on priced reservation units where subvention of the price can be applied for

In theory the UI would also support other sets too, so maybe the current code is desirable, but at least FYI those are the combos that in practice exist and are dealt with.
Implemented that way for historical reasons, maybe would now be done some other way in light of current information.
Actual address fields ended up not being asked on any form currently. But `home_city` is separate from those (in practice the choices are HELSINKI vs OTHER, just to make the distinction between those living there and everyone else).

Reply:
Added these combinations to the created sets. Also made sure that the variant 3 and 4 are only use id said field is true.